Specs/Scene/ShadowMapSpec.js

```diff
@@ -788,16 +818,38 @@ describe(
 
       // Render with shadows
       scene.shadowMap.enabled = true;
+      let shadowedColor;
       renderAndCall(function (rgba) {
+        shadowedColor = rgba;
         expect(rgba).not.toEqual(backgroundColor);
         expect(rgba).not.toEqual(unshadowedColor);
+
+        // The floor is red, and when shadowed, the red
+        // component should be darker
+        const shadowedRed = rgba[0];
+        const unshadowedRed = unshadowedColor[0];
+        expect(shadowedRed).toBeLessThan(unshadowedRed);
       });
 
       // Change the light so that the shadows are no longer pointing straight down
       scene.light = new DirectionalLight({
         direction: lightDirectionAngle,
       });
-      renderAndExpect(unshadowedColor);
+
+      // Change the light direction so that the shadows are no longer pointing straight down
+      renderAndCall(function (rgba) {
+        expect(rgba).not.toEqual(backgroundColor);
+        expect(rgba).not.toEqual(shadowedColor);
+        expect(rgba).not.toEqual(unshadowedColor);
+
+        // After changing the light direction, the floor will appear
+        // a bit darker, but not as dark as when it was shadowed by the box
+        const red = rgba[0];
+        const unshadowedRed = unshadowedColor[0];
+        const shadowedRed = shadowedColor[0];
+        expect(shadowedRed).toBeLessThan(red);
```

Comment:
same suggestion as above

```suggestion
        expect(red).toBeGreaterThan(shadowedRed);
```